### ISSUE TITLE
OpenUri.sublime-settings doesn't work outside the User directory

### ISSUE THREAD

**Issue by john-cj** [jfcherng/Sublime-OpenUri#6]
For personal reasons, I prefer to keep settings for installed packages in separate directories below the User directory, by using the `zz` prefix.

```
Packages
|-- User
|-- zz_overwriting-default-package
|-- zz_overwriting-other-packages
    |-- aaa
    |-- bbb
    |-- openuri
        |-- Default.sublime-mousemap
        |-- OpenUri.sublime-settings
```

**OpenUri.sublime-settings**:

```
{
  "show_open_button": "never",
}
```

This idea works for every other package I have, but not for OpenUri. For some reason, `OpenUri.sublime-settings` _must_ be inside the User directory. Is it a bug or a valid improvement request? (And why it happens?)

**Comment by jfcherng** [comment]
Works for me.


https://user-images.githubusercontent.com/6594915/126856067-f4fad6f2-24cb-4f51-a65b-fee9fe39be8b.mp4

**Comment by jfcherng** [comment]
Fwiw, I think people call the `sublime.load_settings(base_name)` API to load settings. Even if there is a bug, it's ST's.


> Loads the named settings. The name should include a file name and extension, but not a path. The packages will be searched for files matching the base_name, and the results will be collated into the settings object. Subsequent calls to load_settings() with the base_name will return the same object, and not load the settings from disk again.

By design, this plugin only check/update the current activated view. This is most likely why you think it doesn't work.
I've rolled out `7.0.1` to handle that confusion.  https://github.com/jfcherng-sublime/ST-OpenUri/releases/tag/4105-7.0.1

**Comment by john-cj** [comment]
Strange. After I have rebooted the computer, the problem seems to be gone, even thought the package version is still 7.0.0.

One possible reason could be that my Sublime Text (4.1.1.3) is 32-bit. Although 32-bit versions are available to download from the official site if you "hack" the URL, it seems they are not officially supported and so may have some additional bugs.

It's up to you, of course, but you might consider to cancel / revert the 7.0.1 release, since the issue was caused by some another reason and I think it would be better to avoid performance hit, even the minor one, so 7.0.0 is perfectly OK for now.

And thanks for this package and your help. :+1:
